wit_dashboard/wit-dashboard.html

```diff
@@ -7893,6 +7901,23 @@ <h2>Create similarity feature</h2>
           return inferences[0];
         },
 
+        combineExtraOutputs_: function(extraOutputs) {
+          for (let i = 1; i < extraOutputs.length; i++) {
```

Comment:
Could you add a comment here on what i, j and keys mean for future reference?

Reply:
done